Dependencies/d3dx.ini

```diff
@@ -343,8 +344,21 @@ analyse_options = log dump_rt_jps clear_rt
 [Device]
 
 ; overrides screen resolution.
-;width=1280
-;height=720
+; if upscaling is on the resizing functionality is disabled
+; however if upscaling is not zero then you HAVE to specify the resolution the game will be upscaled to
+; to achive the best visual result use the native resolution of your tv or monitor
+;width=2560
+;height=1440
+
+; (0) - disable upscaling; (1) - enable upscaling and allows the game to disable and enable fullscreen mode
+; (2) - enable upscaling and dont allow the game to switch fullscreen mode (always force the fullscreen)
+;upscaling = 1
+
+; upscale mode = 0 migoto create a texture and pushes it as the back buffer to the game seems to work with only few games but 
+; everything seems to run a bit smoother
+; upscale mode = 1 migoto creates second swap chain and pushes it to the game it to the game seems to work with most games
+; dont forget to activate upscaling shader in [present] section
+;upscale_mode = 1
```

Comment:
We are really sloppy with it, but our goal for this file is to have the default options not commented out.  
So I'd change this to be "upscale_mode=0" as the default.